velox/functions/sparksql/MonotonicallyIncreasingId.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/functions/Macros.h"
+
+namespace facebook::velox::functions::sparksql {
+
+template <typename T>
+struct MonotonicallyIncreasingIdFunction {
+  static constexpr bool is_deterministic = false;
+
+  void initialize(const core::QueryConfig& config) {
+    count_ = (int64_t)config.sparkPartitionId() << 33;
```

Comment:
It looks like this functions fails if partition id not set. Maybe document this behavior as well.

Reply:
Added, thanks.